neo4j/io/__init__.py

```diff
@@ -754,10 +754,13 @@ def update_routing_table_from(self, *routers, database=None,
         """
         log.debug("Attempting to update routing table from {}".format(", ".join(map(repr, routers))))
         for router in routers:
-            new_routing_table = self.fetch_routing_table(
-                address=router, timeout=self.pool_config.connection_timeout,
-                database=database, bookmarks=bookmarks
-            )
+            try:
+                new_routing_table = self.fetch_routing_table(
+                    address=router, timeout=self.pool_config.connection_timeout,
+                    database=database, bookmarks=bookmarks
+                )
```

Comment:
Make this method always return `None` instead of raising an exception if anything goes wrong.